.changeset/lucky-mice-live.md

```diff
@@ -0,0 +1,5 @@
+---
+"@atproto/bsky": patch
+---
+
+add feedgen content mode lexicon spec
```

Comment:
Can we add a changeset for `@aproto/api` too, so that it gets marked for a release?